students/cfarris/lessons/lesson03/assignment/customer_model.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env Python3
+
+"""
+    Simple database example with Peewee ORM, sqlite and Python
+    Here we define the schema
+    Use logging for messages so they can be turned off
+
+"""
+import logging
+from peewee import *
+
+logging.basicConfig(level=logging.INFO)
```

Comment:
You don't have to set the basicConfig more than once, and probably better not to.